src/components/numbered-code-snippet/show-hide-lines.js

```diff
@@ -3,7 +3,7 @@ import PropTypes from 'prop-types';
 import Icon from '@mapbox/mr-ui/icon';
 
 const containerClasses =
-  'w-full bg-gray-faint color-gray-dark color-gray-on-hover pl12 py6 cursor-pointer txt-s';
+  'w-full bg-gray-lighter color-gray-dark color-text-on-hover pl12 py6 cursor-pointer txt-s';
```

Comment:
Good instinct on updating these colors, what do you think about leaning into the blue instead? I think blue has a greater contrast and would match the left-side blue border. (Unfortunately `color-text-on-hover` is not a valid Assembly class.)

```suggestion
  'w-full bg-blue-faint color-blue-dark color-blue-on-hover pl12 py6 cursor-pointer txt-s';
```

If you like this idea, we could probably remove `bg-darken5` from the two buttons in the file, too.